daffodil-tdml-processor/src/main/scala/org/apache/daffodil/tdml/SchemaCache.scala

```diff
@@ -45,9 +45,9 @@ object SchemaDataProcessorCache extends SchemaCache[(Seq[Diagnostic], DFDL.DataP
 class SchemaCache[CachedType, DiagnosticType] {
 
   private class Cache
-    extends mutable.HashMap[(URISchemaSource, Boolean, Boolean, Option[String], Option[String]), (URISchemaSource, CachedType)] {
+    extends mutable.HashMap[(URISchemaSource, Boolean, Boolean, Option[String], Option[String], Map[String, String]), (URISchemaSource, CachedType)] {
```

Comment:
Lines too long. Add comments explaining why the tunables map is now included. Comments per member of this hashmap key tuple.